WebRoot/css/login.css

```diff
@@ -520,4 +520,41 @@ label#remember {
     height: 15px;
     padding: 0;
     background-size: 15px 15px;
+}
+
+.errorBox {
+	background: #fff;
+	border-radius: 3px;
+	max-width: 400px;
+	margin: 10% auto;
+	padding: 24px;
+	box-shadow: 0 5px 8px rgba(0,0,0,0.3);
+}
+
+.errorBox h2 {
+	margin: 0 0 24px;
+	font-size: 20px;
+	font-weight: normal;
+}
+
+.errorBox p {
+	font-size: 13px;
```

Comment:
aren't we using legacy defined font size variables?

Reply:
have used a variable whose font-size is almost matching 13px